### PR TITLE
Add Docker compose for SQL Server local development

### PR DESCRIPTION
**Pull request by aaronamm**
- This PR is for simplify a local development by adding Docker compose file for creating local SQL Server instance.
- Just follow steps in README, "SQL Server for Linux on Docker container" section.
- Then you can use a database in SQL Server instance to setup your OrchardCore website.

### CODE REVIEW

**Comment by Skrypt** [comment]
Everyone has their own docker-compose.yml file for their own projects. This would be more appropriate to add this to a documentation with examples.

File: README.md [hunk 1]
```diff
@@ -55,6 +55,17 @@ Here is a more detailed [roadmap](https://github.com/OrchardCMS/OrchardCore/wiki
 Docker images and parameters can be found at <https://hub.docker.com/u/orchardproject/>  
 See [Docker documentation](https://docs.docker.com/engine/reference/commandline/run/#publish-or-expose-port--p---expose) to expose different port.
 
+### SQL Server for Linux on Docker container
+
+- You can use SQL Server for Linux with Docker Linux container.
+- To lanuch a SQL Server instance,
```
Comment (agriffard) [inline README.md:61]: lanuch => launch

File: README.md [hunk 2]
```diff
@@ -55,6 +55,17 @@ Here is a more detailed [roadmap](https://github.com/OrchardCMS/OrchardCore/wiki
...
 See [Docker documentation](https://docs.docker.com/engine/reference/commandline/run/#publish-or-expose-port--p---expose) to expose different port.
 
+### SQL Server for Linux on Docker container
+
+- You can use SQL Server for Linux with Docker Linux container.
+- To lanuch a SQL Server instance, 
+    - Create `sa_password.secret` file at root level of the project
+    - Put your SA password in the file and make sure it uses LF line feed.
+    - Run `docker compose up`
+    - Connect to the database server with `localhost, 1433`, SQL Server Authentication as `SA` and  your SA password.
```
Comment (agriffard) [inline README.md:65]: and  your SA password. => Remove extra space

**Comment by deanmarcussen** [comment]
This would be much better suited as a guide, because it is not about running orchard core on docker, it's about how to setup a sql server instance via docker.

I don't think we should be including docker-compose files in the repository directly.

(They're too opinionated as to database choice, how to best use docker etc... )

**Comment by aaronamm** [comment]
@Skrypt @deanmarcussen Thank you for your suggestion. I agree with you.
However, for adding it as a guide, could you please suggest me where should I put it?

Here is what I think:
- Starting the content from [https://github.com/OrchardCMS/OrchardCore/blob/dev/src/docs/guides/README.md](https://github.com/OrchardCMS/OrchardCore/blob/dev/src/docs/guides/README.md) in `Orchard Core CMS Guides` section.
- And put the main content with code example in https://github.com/OrchardCMS/OrchardCore/blob/dev/src/docs/guides/sql-server-docker/README.md

Thanks.

**Comment by deanmarcussen** [comment]
It actually looks like this is all covered in a recent bit of docs from @Skrypt here https://docs.orchardcore.net/en/dev/docs/topics/docker/

**Comment by aaronamm** [comment]
@deanmarcussen  Great, thank you so much. 
I haven't known we have it in the document.

I will find if there is some think that I can add to this document https://github.com/OrchardCMS/OrchardCore/tree/dev/src/docs/topics/docker
and open a new PR.

Therefore, I will close this PR CC @Skrypt .

Thanks

**Comment by Skrypt** [comment]
Don't close PR yet there are interesting things in this PR.

**Comment by aaronamm** [comment]
@Skrypt  could you please let me know what should I do with this PR?
Here is what I think.
- Close this PR.
- Move some useful content and put it in `https://github.com/OrchardCMS/OrchardCore/blob/dev/src/docs/topics/docker/README.md`

Thanks.

**Comment by Skrypt** [comment]
Okay, or simply update this PR to do that. And rename it's title. Simpler.

**Comment by aaronamm** [comment]
@Skrypt  That's a good idea and we also keep this useful conversation. 
However, for a code change, I will force push and replace existing commit.

Thanks, everyone.
@Skrypt @deanmarcussen  @agriffard

**Comment by Skrypt** [comment]
Closing, no update.

**Pull request closed without merging**